docs/core-concepts/core-sdk/cli/dotnet-publish.md

```diff
@@ -0,0 +1,81 @@
+dotnet-publish
+==============
+
+## NAME
+
+`dotnet-publish` - packs the application and all of its dependencies into a folder getting it ready for publishing
+
+## SYNOPSIS
+
+dotnet-publish [--framework]  
+    [--runtime] [--build-base-path] [--output]  
+    [--version-suffix] [--configuration]  
+    [< project >]  
+
+## DESCRIPTION
+
+`dotnet-publish` compiles the application, reads through its dependencies specified in `project.json` and publishes the resulting set of files to a directory. 
+
+Depending on the type of portable app, the directory contains the following:
+
+1. **Portable application** - application's intermediate language (IL) code and all of application's managed dependencies.
+    * **Portable application with native dependencies** - as above with a sub-directory for each native dependencies' 
+    supported platform. 
+2. **Self-contained application** - as above as well as the entire runtime for the targeted platform.
+
+The above types are covered in more details in the [types of portable applications](../../app-types.md) document. 
+
+## OPTIONS
+
+`[project]` 
+    
+`dotnet-publish` needs access to `project.json` to work. If it is not specified on invocation via [project], 
+`project.json` in the current directory will be the default.     
+If no `project.json` can be found, `dotnet-publish` will error out. 
+
+`-f`, `--framework` [FID]
+
+Publish the application for a given framework identifier (FID). If not specified, FID is read from `project.json`. In case of no valid framework found, the command will error out. In case of multiple valid frameworks found, the command will publish for all valid frameworks. 
```

Comment:
In case of no valid framework found -> If no valid framework is found
In case of multiple valid frameworks found -> If multiple valid frameworks are found
I'd also avoid future tense in the topics, but this could be something to improve later on.